resources/views/Widgets/Common/LinkWidget.json

```diff
@@ -23,7 +23,7 @@
                 "tooltipText": "Widget.linkSizeTooltip",
                 "listBoxValues": [
                     {
-                        "value": "btn-sm",
+                        "value": "sm",
```

Comment:
@Tim-M-S Auf welche Schreibweise hatten wir uns hier final geeinigt? Wollten wir nicht "btn-sm" als Wert lassen, um für "Medium" kein Klasse "btn-" zu erzeugen?

Reply:
Oh, das war mir unbekannt. Mein letzter Stand vom @Tim-M-S war, nur sm und lg zu übergeben. Im Twig wurde das vorherige zu btn-btn-sm umgewandelt, weshalb ich hier einen  Fehler beim händischem Merge vemutet hab.

Reply:
Klein: "btn-sm"
Standard: ""
Groß: "btn-lg"

Und am Widget {{ buttonSize }} in die classes geben

Reply:
Da hat der Späte vergessen vorher das Twig anzupassen :p

Reply:
Done.